src/nunit.analyzers/ClassicModelAssertUsage/ClassicModelAssertUsageCodeFix.cs

```diff
@@ -121,6 +158,13 @@ ArgumentSyntax CastIfNecessary(ArgumentSyntax argument)
             }
         }
 
+        protected virtual (ArgumentSyntax ActualArgument, ArgumentSyntax ConstraintArgument) UpdateArguments(
+            Diagnostic diagnostic,
+            IReadOnlyDictionary<string, ArgumentSyntax> argumentNamesToArguments)
+        {
+            throw new NotImplementedException($"Class must override {nameof(UpdateArguments)}");
```

Comment:
This causes all the other Classic CodeFixes to fail until they are updated.
Although once we have the framework in place, the update should be straight forward: Replacing indices access with parameter names.

Reply:
@manfred-brands Yup, I'm in the process of updating all other `*CodeFix`es.

However, I'm not sure how many tests should be added to other `*CodeFixTests` files, although I'm sure we need _some_. When you get a chance, could you point out the list of tests that should be ported over (potentially after renaming) or added to other `*CodeFixTests` files?